third_party/libwebrtc/webrtc/modules/congestion_controller/nada_owd_bwe.cc

```diff
@@ -25,51 +25,14 @@
- * for more details.
- */
-constexpr float kNadaBweParamPrio  = 1.0;   // weight of priority for the flow [PRIO: dimensionless]
-constexpr float kNadaBweParamXref  = 10.0;  // reference congestion level  [XREF: in ms]
-constexpr float kNadaBweParamXDefault = 20.0;  // default congestion level [in ms]
-constexpr float kNadaBweParamKappa = 0.5;   // scaling parameter for gradual rate update [KAPPA: dimensionless]
-constexpr float kNadaBweParamEta  = 2.0;  // scaling parameter for gradual rate update [ETA: dimensionless]
-constexpr float kNadaBweParamTau = 500.;  // Upper bound of RTT for gradual rate update [TAU: in ms]
-
-constexpr uint32_t kNadaBweDefaultBitrate = 600000;      // 600 Kbps
-constexpr uint32_t kNadaBweDefaultMinBitrate =  300000;  //  300 Kbps
-constexpr uint32_t kNadaBweDefaultMaxBitrate = 3000000;  // 3000 Kbps
-
-constexpr int64_t kNadaBweParamDeltaMs = 100; // Target interval for feedback and/or rate update [DELTA: in ms]
-constexpr int64_t kNadaBweParamMinDeltaMs =  20; // Minimum value of delta for rate calculation [in ms]
-constexpr int64_t kNadaBweParamMaxDeltaMs = 200; // Maximum value of delta for rate calculation [in ms]
-constexpr int64_t kNadaBweParamLogwinMs  = 500;  // Observation time window for calculating
-constexpr int64_t kNadaBweParamLogwinMs2 = 3000; // Observation time window for long-term minimal baseline forward delay value [in ms]
-constexpr int64_t  kNadaBweDefaultRttMs = 50.;   // in ms
-constexpr int64_t  kNadaBweParamQepsMs = 10;    // Threshold for determining queueing delay build-up [QEPS: in ms]
-constexpr int64_t  kNadaBweParamQboundMs = 50;  // Upper bound on self-inflicted queuing delay [QBOUND: in ms]
-constexpr int64_t  kNadaBweParamDfiltMs = 120;  // Bound on filtering delay [DFILT: in ms]
-constexpr float    kNadaBweParamGammaMax = 0.2;  // Upper bound on rate increase ratio for accelerated ramp-up
-                                                // [GAMMA_MAX: dimensionless]
-constexpr float    kNadaBweParamPlrSmooth = 0.1;  // Smoothing factor for EMA of packet loss ratio
-
-constexpr float    kNadaBweParamQthMs =50.;    // Delay threshold for invoking non-linear warping (QTH: in ms)
-constexpr float    kNadaBweParamLambda =0.5;   // Scaling parameter in the exponent of non-linear warping (LAMBDA: dimensionless)
-constexpr float    kNadaBweParamPlrRef =0.01;  // Reference packet loss ratio (PLRREF: dimensionless)
-constexpr float    kNadaBweParamDLossMs =10.;  // Reference delay penalty for loss when packet loss ratio is at PLRREF 
-                                               // (DLOSS: dimensionless)
-
-constexpr int64_t kDefaultProbingIntervalinMs = 100;  // in ms
-constexpr int64_t kStreamTimeOutMs = 2000;            // in ms
-
-// This ssrc is used to fulfill the current API but will be removed
-// after the API has been changed.
-constexpr uint32_t kNadaOwdFixedSsrc = 0;
+constexpr int kNadaBweDefaultBitrate = 600000;        // Default initial rate 
```

Comment:
Could we define these inside nada_core.h ? (in other words: any reason why they're defined here?)